manifests/m/MailStore/MailStoreHome/25.1.0.22653/MailStore.MailStoreHome.installer.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: MailStore.MailStoreHome
+PackageVersion: 25.1.0.22653
+InstallerType: portable
+Installers:
+- Architecture: x86
+  InstallerUrl: https://download.mailstore.com/dont-link-this/36B44217-2BF0-4603-8160-35DF2A708DC3/25.1.0.22653/MailStoreHomeSetup-25.1.0.22653.exe

```

Comment:
The URL explicitly says to not link this; 
@denelon/@stephengillie - Thoughts on policy? Would this violate MailStore TOS?

Reply:
I don't see this specifically mentioned in the [T&C](https://www.mailstore.com/en/legal/agreement-terms-conditions-license-use-commercial-versions/). But yes it does appear to be a request not to use the URL in this way.